README.md

```diff
@@ -84,24 +102,86 @@ ngrok http 4000
 ```
 
 
+## Use Cases
 
+There are two types of actors in LaraMessaging:
 
+- Account Holder: There is only one account holder. The Account holder can see and read the messages from the anonymous users without knowing who they are.
 
+- Anonymous User: Anonymous users are able to anonymously send maessages to the account holder
+
+For MVP:
+
+- Messages cannot be deleted from prompt by user but may be automatically deleted by the system after a designated length of time. 
+- Anonymous Users once leaving the chat cannot rejoin a chat but must create a new one
+- Anonymous Users do not need to create accounts
+    
+
+#### Simple Anonymous User Interaction
+
+1. Account Holder clicks register an account
+2. Enters email and necessary signup information
+3. Account Holder recieves a room code
+4. Account Holder gives room code to Anonymous Users
+5. Account Holder goes to messages view
+6. Anonymous User visits website
+7. Anonymous User enters room code
+8. Anonymous User is prompted with a conversation view to start a conversation with the Account Holder
+9. Anonymous User sends a message and can see the message is sent
+10. Account Holder recieves the message from an Anonymous User that is given a random name in the conversation box
+11. Account Holder and Anonymous User can send messages to each other
+12. Anonymous User closes the browser or otherwise ends the chat the Account Holder recieves a message in the conversation box saying the anonymous user has left
+
+#### Multiple Anonymous User Interactions
+
+1. Account Holder clicks register an account
+2. Enters email and necessary signup information
+3. Account Holder recieves a room code
+4. Account Holder gives room code to Anonymous Users
+5. Account Holder goes to messages view
+6. Anonymous User visits website
+7. Anonymous User enters room code
+8. Anonymous User is prompted with a conversation view to start a conversation with the Account Holder
+9. Anonymous User sends a message and can see the message is sent
+10. Account Holder recieves the message from an Anonymous User that is given a random name in the conversation box
```

Comment:
Account holder should have a list of chats and be able to select the random chat from a list. I envision a UI similar to android messages web view